src/test/shell/testenv.sh

```diff
@@ -501,7 +507,6 @@ function assert_bazel_run() {
 }
 
 setup_bazelrc
-setup_clean_workspace
```

Comment:
Why removing this?

Reply:
Because it's also called again in `create_and_cd_client` here: https://github.com/bazelbuild/bazel/pull/5387/files/ea9aaba28fff52d3332c07bc6c3ad444c4910f38#diff-c467ef71bdf4e156e40ab02be09dccd5R540

Reply:
Thanks for explaining!